### ISSUE TITLE
TypeError: websites is undefined

### ISSUE THREAD

**Issue by FromBabylon** [seeker89/cozy-pfm#45]
I have a problem while adding bank "Société Générale". Web console shows "TypeError: websites is undefined" as an error. I also have this error for others bank like BNP Paribas, American Express... although Caisse Epargne works well. 
Indeed I don't see any websites in the files. 

How to fix it ?

**Comment by FromBabylon** [comment]
Is it a bug or just that websites are really undefined ?
I cannot see any websites in files. Only for Banque Pop & Crédit Agricole. How is the app supposed to retrieve bank access ?

**Comment by jsilvestre** [comment]
Hi @FromBabylon , thank you for the report.

A member from the community, @krichtof recently added new banks that can have various websites, depending on the area you lived in (regionalized banks). I bet he didn't test the case for classic banks (la Société Générale is one of them) ;)

So it's more about fixing the case where there is no "websites", rather than trying to find something that (like you figured) doesn't exist.

I will try to fix it quickly.

**Comment by jsilvestre** [comment]
@FromBabylon Can you please update your app and tell me if the problem has been fixed? Thank you and sorry for the inconvenience.

**Comment by Lich4r** [comment]
There is nothing you should be sorry about.
It still doesn't work. The "Website is undefined" error isn't here though. I now have "HTTP/1.1 500 Internal Server Error" when trying to reach https://MyCozyIP/apps/pfm/bankaccesses according to Firefox web console. 
I've checked again with Caisse Epargne it works well but I got the same Internal error if I make a mistake in the password. I am sure I have made no mistakes with SOG password so it is strange here :)

**Comment by frankrousseau** [comment]
Thank for your feedback @FromBabylon.

@jsilvestre Thank you for taking care of that issue but don't close issues until the reporter confirmed that the problem is fixed. I'm going to modify the [process](https://github.com/cozy/cozy-guidelines/blob/master/process/bug-report.md).

**Comment by jsilvestre** [comment]
Yeah, it automatically closed itself from the PR, actually, my bad.

**Comment by jsilvestre** [comment]
@FromBabylon Could you update the indexer? The connectors were not up-to-date (the modules that handles the data retrieval).
If you can't (i.e. you don't know what I'm talking about): do you host your Cozy yourself or is it hosted by Cozycloud?

**Comment by Lich4r** [comment]
I host cozy myself. I've just had a big problem with my server so I had to reinstall everything. 
It is now working ! Awesome ! Thanks a lot. I can add SOG account without problem. Checking account Balance seems correct however I see 0€ as Credit card balance which is wrong (this should be negative). Looks like pfm couldn't get all the datas and credit card account doesn't show any operation after 27/12/2014. Any idea ? 

Also now in my cozy home I see "beta" aside the cloud icon at the top left corner. Is it normal ?

**Comment by jsilvestre** [comment]
PFM: do you have any account that is not something like "Compte courant", "PEL", "Livret A"? I'm talking about "compte titre" and that kind of account. They are not supported, and the only solution is to remove them from the account screen.
The fact you don't have data after 27/12/2014 is kind of weird...no idea here.

Beta indicator: It is! We added that indicator so users are not too confused when something doesn't work. We are still in beta, after all.

**Comment by Lich4r** [comment]
I have one Checking Account (compte courant) which works well and one credit card account  (Compte carte débit différé) for which I have no data after 27/12. 
Why are they not supported ? I mean they work the same way. 

Got it ;)

**Comment by jsilvestre** [comment]
It's just weird, both accounts are supposed to work. Can you try to remove the credit card account and "refresh" the data?

**Comment by Lich4r** [comment]
If I remove my credit cart account and then add it again I have data untill 15/01 ^^ But balance stille at 0€. 
(checking account still working well)

**Comment by jsilvestre** [comment]
Since it works well for me (Société Générale) I'm afraid I won't be able to help you more, because I won't ask for all your banking operation data...If you are willing to dig by yourself, I'd gladly help. I suggest you install "Kresus" application which work much better. Kresus still requires "PFM" for data importation, but the app itself is better. You can find the repository here: https://github.com/bnjbvr/kresus. It's done by a community member, @bnjbvr!

**Comment by Lich4r** [comment]
I use Kresus but datas come from pfm so same problem for credit card account. 
Are you sure Credit Card Account are supported ? ("Compte Carte débit différé"). Those accounts wipe each month by a debit on the checking account for the current balance of credit card account. I can't see the "wipe transaction" for the past month. I have to say that this transaction doesn't look the same than others on my bank website so maybe it is not supported here ?

There is no errors when refreshing datas.

**Comment by jsilvestre** [comment]
From what you say, it might be an unsupported account. You can try remove this account, and only this account, then reload the page and see if it's better.
